lib/tasks/simplecov_check_coverage.rb

```diff
@@ -0,0 +1,15 @@
+require 'simplecov'
+result_hash = SimpleCov.result.to_hash
+puts "#{result_hash.keys}"
+puts "NOW CHECKING COVERAGE"
+puts "Coverage = #{SimpleCov.result.covered_percent}"
```

Comment:
@HalahAb , this is looking good so far. One CircleCI feature you may find useful, if you haven't already been using it, is the "Rerun job with SSH" button. You can throw a `binding.pry` line into your code, then run the job with SSH so that you can tinker with the code directly on the CircleCI server, look around the environment, etc.

Reply:
@MothOnMars Thanks Martha! oh yeah that would be very useful, I saw that button but I'm not sure I can do anything with it, it's telling me I require write permissions to trigger builds.

Reply:
Ah, right. Phooey.